plugins/libvirt/libvirt_test.go

```diff
@@ -0,0 +1,24 @@
+package libvirt
+
+import (
+	"testing"
+
+	"github.com/influxdb/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLibvirt(t *testing.T) {
```

Comment:
The actual test does not need a running libvirt. But the compilation of go-libvirt needs the libvirt headers (libvirt-dev package). I added that package to `circle.yml`. Should I document that dependency anywhere else? 
For OS X there is a Homebrew formula for libvirt (`brew install libvirt`) which I have installed. That's why I didn't notice that fact earlier.